src/coreclr/src/ilasm/asmman.cpp

```diff
@@ -965,6 +965,7 @@ HRESULT AsmMan::EmitManifest()
                 {
                     wcscpy_s(wzFileName,2048,pwzInputFiles[j]);
                     pwz = wcsrchr(wzFileName,'\\');
+                    if(pwz == NULL) pwz = wcsrchr(wzFileName,'/');
                     if(pwz == NULL) pwz = wcsrchr(wzFileName,':');
```

Comment:
We should not be checking for `:` on Unix at all. Can you please add #ifdef TARGET_WINDOWS around  this line?

Reply:
I've also done this where relevant.